migrations/20190430131520_add_job_group.sql

```diff
@@ -0,0 +1,24 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+CREATE TABLE "job_groups" (
+  "id" uuid DEFAULT uuid_generate_v4() UNIQUE,
+  "app_id" uuid NOT NULL,
+  PRIMARY KEY ("id")
+);
+
+ALTER TABLE "jobs"
+ADD CONSTRAINT jobs_group_app_id_apps_id_foreign
+FOREIGN KEY (app_id)
+REFERENCES apps(id)
+ON DELETE CASCADE
+ON UPDATE CASCADE;
+
+ALTER TABLE "jobs" ADD COLUMN job_group_id uuid DEFAULT uuid_generate_v4();
```

Comment:
Not all jobs should have a job_group, right? Shouldn't we keep this `null` for most of the cases and only set a `job_group_id` when necessary?

Reply:
The idea of the jobs groups is to join jobs when showing than in the user interface of the project.
Because of that, I generate an id for the previous jobs. If I keep this field `null`, the UI will need to treat this as a special case.

Reply:
Okay, we talked about what you plan to do in the future and we'll go with this for now.